### DISCUSSION TITLE
Working with product line items & inventory?

### DISCUSSION THREAD

**Discussion by ClaytonFarr** [Budibase/budibase#4334] [Help]
I'm getting started trying Budibase and playing around with modeling data for a few specific use cases. I've looked through the docs and discussions to try and find more detail on this, but have come up short on a couple of things –

1. Is it possible to relate multiple items to another item and also edit the values of the related items (e.g. line items that have quantity set for each item)?
2. What's the right way to model items to have values that will be updated by other items (e.g. product inventory)?

For example, a basic store or rental app –

- A store has pre-defined products (with current inventory, unit price, etc.)
- Orders are created with 1+ products, with the ability to set a quantity for each of them
- A product's inventory count is updated as orders take or release inventory

If this is possible, what would be the correct way to create to model the data for the products and/or orders? 

Thanks for any help.

**Comment by melohagan** [comment]
Hey @ClaytonFarr 

Sorry for the late reply, but may help:

- https://docs.budibase.com/docs/form-steps
- https://docs.budibase.com/docs/chaining-actions#example-creating-and-linking-two-table-entries-in-one-form

**Comment by ClaytonFarr** [comment]
Thanks @melohagan
